src/main/java/com/volasoftware/tinder/utils/BotInitializer.java

```diff
@@ -46,8 +47,26 @@ public class BotInitializer {
 
     private final NameGenerator nameGenerator;
 
+    private final JdbcTemplate jdbcTemplate;
+
     @PostConstruct
     void initialize() {
+        setupBots();
+    }
+
+    private void buildInitialAccounts() {
+        String insertAccount1 = "insert into account(first_name, last_name, email, password, gender, account_type, created_at, updated_at, is_verified) values (?, ?, ?, ?, ?, ?, NOW(), NOW(), ?)";
```

Comment:
You could just use the AccountRepository and create new accounts, without using jdbcTemplate and raw sql queries. Maybe I don't get the whole idea.